api/openapi/common.yaml

```diff
@@ -0,0 +1,34 @@
+openapi: 3.0.3
+info:
+  title: Common types of Turing schemas
+  version: 0.0.1
+
+paths:
+
+components:
+  schemas:
+    Id:
+      type: "integer"
+      format: "int32"
+
+    FieldSource:
+      type: "string"
+      enum:
+        - "header"
+        - "payload"
+
+    pagination.Paging:
```

Comment:
Why is this schema named like this, instead of the old `PaginatedResults` ?

Reply:
`PaginatedResults` is more like a composite type of `paging` (information about current page number and total number of pages) + `results` (array with data objects). This `pagination.Paging` is just a schema of a former.

But if the question is more about using this `pagination.` prefix, then I'm just experimenting with it to achieve a better grouping of types between packages. (There is a chance that this will be changed later)